src/components/expanded-state/UniqueTokenExpandedState.tsx

```diff
@@ -274,17 +274,8 @@ const UniqueTokenExpandedState = ({ asset: passedAsset, external }: UniqueTokenE
     }
   }, [isReportSpamToastActive]);
 
-  const {
-    collection: { description: familyDescription, external_url: familyLink, slug },
-    description,
-    familyImage,
-    familyName,
-    isSendable,
-    marketplaceName,
-    traits,
-    uniqueId,
-    fullUniqueId,
-  } = asset;
+  const { description, familyImage, familyName, isSendable, marketplaceName, traits, uniqueId, fullUniqueId } = asset;
+  const { description: familyDescription, external_url: familyLink, slug } = asset.collection || {};
```

Comment:
can't you just default the collection itself to `{}` and deconstruct in one line still?

```
const {
    collection: { description: familyDescription, external_url: familyLink, slug } = {},
    description,
    familyImage,
    familyName,
    isSendable,
    marketplaceName,
    traits,
    uniqueId,
    fullUniqueId,
  } = asset;
```